cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -78,6 +78,49 @@ public List<Parameter> getParameters() {
     public void export(Network network, Properties parameters, DataSource dataSource, ReportNode reportNode) {
         Objects.requireNonNull(network);
 
+        CgmesExportContext context = createContext(network, parameters, reportNode);
+
+        // Export the network
+        if (Parameter.readBoolean(getFormat(), parameters, CGM_EXPORT_PARAMETER, defaultValueConfig)) {
+            exportCGM(network, dataSource, context);
+        } else {
+            exportIGM(network, dataSource, context);
+        }
+    }
+
+    public void createDefaultControlAreaInterchange(Network network) {
+        createDefaultControlAreaInterchange(network, null);
+    }
+
+    public void createDefaultControlAreaInterchange(Network network, Properties parameters) {
+        Objects.requireNonNull(network);
+
+        CgmesExportContext context = createContext(network, parameters, ReportNode.NO_OP);
+
+        String controlAreaId = context.getNamingStrategy().getCgmesId(refTyped(network), CONTROL_AREA);
+        Area area = network.newArea()
+                .setAreaType("ControlAreaTypeKind.Interchange")
```

Comment:
`ControlAreaTypeKind.Interchange` enum value is used here as well as in:
- `EquipmentExport`
- `SteadyStateHypothesisExport`
- `ControlAreaEq`

I think it would be worth creating a constant in `CgmesNames`.